lib/Distribution/Sle/15_current.pm

```diff
@@ -30,6 +30,8 @@ use Installation::SystemRole::Sle::SystemRoleController;
 use Installation::ModuleRegistration::SeparateRegCodesController;
 use YaST::DNSServer::Sle::DNSServerController;
 use YaST::DNSServer::Sle::DNSServerSetupController;
+use Installation::NtpConfiguration::NtpConfigurationPage;
+use YuiRestClient;
```

Comment:
you don't need this, you can inherit lib/Yam/PageBase.pm in corresponding Page

Reply:
`app => YuiRestClient::get_app()}` is not needed if you use that parent class.

Reply:
I need this at here.  As we removed controller part. At here, we need get app then to get the widget information.
if remove it, it will get error: https://openqa.suse.de/tests/11848431

Reply:
you shouldn't get that issue if you add `use parent 'Yam::PageBase';` in Page, see for example ControlCenterPage.pm